src/unitxt/metric_utils.py

```diff
@@ -320,6 +320,30 @@ def prepare(self):
 )
 
 
+class EvaluationResults(list):
+    @property
+    def score(self):
```

Comment:
I think "score"  is a problematic name (should have been "scores"). Do we want to start change it here.    We could "global_scores", "group_scores" and "subset_scores", "instance_scores" attributes.  Slowly we will move off the list and use only the attributes.

Reply:
I think the most useful should be the shortest most natural name. So assuming it is used like this: results.score the most natural way is to call it something like results.summary

The slow movement from list can start by using to_list() for most cases and examples that actually access the list.

Reply:
It worth while to discuss the names, as they stay with us for a long time.  

"result.score" sounds like it returns a single number. Note the result also contains instances which are not only score.

result.scores -
result.subset_scores 
result.group_scores
result.instances - (prediction, references, task  data, and scores)